tools/chplcheck/src/chplcheck.py

```diff
@@ -137,9 +137,18 @@ def apply_edits(edits: List[Edit], suffix: Optional[str]):
         edits.sort(key=lambda f: f.start, reverse=True)
         with open(file, "r") as f:
             lines = f.readlines()
+
+        prev_start = None
         for edit in edits:
-            (line_start, char_start) = edit.start
-            (line_end, char_end) = edit.end
+            start = edit.start
+            line_start, char_start = edit.start
+            end = edit.end
+            line_end, char_end = edit.end
+
```

Comment:
The duplicated unwrapping of `edit.start` and `edit.end` is unfortunate, maybe replace the uses of `line_start` with `start[0]`, etc?

Reply:
I think there's too many uses of `start[0]` to make that reasonable. I'd rather unpack _and_ keep the top-level variable.

Reply:
In that case, I would replace uses  of `start` with `edit.start`. I see only 1 usage. Same with `end`